recipe/meta.yaml

```diff
@@ -1,75 +1,93 @@
 {% set name = "bentoml" %}
-{% set version = "0.13.1" %}
+{% set version = "1.0.0" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/BentoML-{{ version }}.tar.gz
-  sha256: 2f53bd2b659e305a97f5cb46fe97f92f65416959e29199fdc55b7637bf7f247b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bentoml-{{ version }}.tar.gz
+  sha256: 6bc4a4bb8f53b6ee70d1169ee439d89680f96771fbb3f11fb5eba13f96598c88
 
 build:
   number: 0
   noarch: python
   entry_points:
-    - bentoml=bentoml:commandline_interface
+    - bentoml=bentoml._internal.cli:cli
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
     - python >=3.7
+    - setuptools >=59.0
+    - setuptools-scm >=6.2.3
+    - wheel  
   run:
     - python >=3.7
-    - aiohttp
-    - aiohttp-cors ==0.7.0
-    - alembic
-    - boto3
-    - cerberus
-    - certifi
+    - aiofiles
+    - aiohttp <=3.8.1
+    - attrs >=21.1.0
+    - backports.cached-property
+    # NOTE: cattrs has not been updated (last version: 1.10.0)
+    # - cattrs >=21.1.0
+    - cattrs >=1.10.0
```

Comment:
This means that bentoml won't work on conda, because the latest version of cattrs requires cattrs to be >=21.1.0

Reply:
Please see my comments [here](https://github.com/conda-forge/bentoml-feedstock/pull/4#issuecomment-1186130474) and [here](https://github.com/conda-forge/bentoml-feedstock/pull/4#issuecomment-1186107032).

Line number `35` is used to make sure that the rest of the packages lead to a successful build of zenml. This PR cannot be merged until `cattrs >=22.1.0 is available.

Also note: it is a typo in your requirements in zenml. There is no `cattrs` version `21.1.0` available on PyPI. Please correct me, if I am mistaken here. Otherwise, please correct the requirement.